simulation_run.py

```diff
@@ -224,15 +224,42 @@
 
     start = timer()
     tmcs_results = contributivity_measures.truncated_MC(current_scenario,
-      sv_accuracy=0.01, alpha=0.9, contrib_accuracy=0.05)
+      sv_accuracy=0.01, alpha=0.95, truncation=0.05)
     end = timer()
 
-    tmcs_contrib = contributivity.Contributivity('TMCS values', tmcs_results['sv'], tmcs_results['std_sv'], np.round(end - start))
+    tmcs_contrib = contributivity.Contributivity('TMC values', tmcs_results['sv'], tmcs_results['std_sv'], np.round(end - start))
 
     current_scenario.append_contributivity(tmcs_contrib)
     print('\n## Evaluating contributivity with Truncated Monte Carlo Shapley (TMCS):')
     print(tmcs_contrib)
 
+    #%% Contributivity 4:   Monte Carlo Shapley
+
+    start = timer()
+    mcs_results = contributivity_measures.truncated_MC(current_scenario,
+      sv_accuracy=0.01, alpha=0.95, truncation=0.00)
+    end = timer()
+
+    mcs_contrib = contributivity.Contributivity('MC values', mcs_results['sv'], mcs_results['std_sv'], np.round(end - start))
+
+    current_scenario.append_contributivity(tmcs_contrib)
+    print('\n## Evaluating contributivity with  Monte Carlo Shapley :')
+    print(mcs_contrib)
+
+
+
+    #%% Contributivity 5: importance sampling
+
+    start = timer()
+    IS_results = contributivity_measures.IS(current_scenario,
```

Comment:
It crashes on my machine: there is `IS()` method in `contributivity_measures`.

Reply:
Do you have an error message? Or is it a full crash?

Reply:
Ok I had just forgot to update the simulation_run file. It should work now.